### PR TITLE
Create json:api backend -- Organizations/Users

### PR DESCRIPTION
**Pull request by chrisvire**
WIP: Don't Merge

### CODE REVIEW

File: source/OptimaJet.DWKit.StarterApplication/Controllers/OrganizationInvitesController.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using JsonApiDotNetCore.Controllers;
+using JsonApiDotNetCore.Services;
+using Microsoft.Extensions.Logging;
+using Optimajet.DWKit.StarterApplication.Models;
+
+namespace Optimajet.DWKit.StarterApplication.Controllers
+{
+    public class OrganizationInvitesController : JsonApiController<OrganizationInvite>
```
Comment (NullVoxPopuli) [inline source/OptimaJet.DWKit.StarterApplication/Controllers/OrganizationInvitesController.cs]: Music to my eyes! :+1:

I love inheriting default REST/CRUD

File: source/OptimaJet.DWKit.StarterApplication/Controllers/UsersController.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using JsonApiDotNetCore.Controllers;
+using JsonApiDotNetCore.Services;
+using Microsoft.Extensions.Logging;
+using Optimajet.DWKit.StarterApplication.Models;
+
+namespace Optimajet.DWKit.StarterApplication.Controllers
+{
+    public class UsersController : JsonApiController<User>
```
Comment (NullVoxPopuli) [inline source/OptimaJet.DWKit.StarterApplication/Controllers/UsersController.cs]: we may want to add a "current user" action here

File: source/OptimaJet.DWKit.StarterApplication/Optimajet.DWKit.StarterApplication.csproj [hunk 3]
```diff
@@ -28,8 +29,12 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.1.1" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.1.1" />
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.1.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.1.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.1.0-preview1-final" />
```
Comment (NullVoxPopuli) [inline source/OptimaJet.DWKit.StarterApplication/Optimajet.DWKit.StarterApplication.csproj]: woah, preview?! livin on the edge now!

**Comment by chrisvire** [comment]
I was waiting to add changes to ./run for updating database and running 'EF migrations' commands.

**Pull request merged**